### PR TITLE
feat: flatpak support

### PR DESCRIPTION
**Pull request by wiiznokes**
update the manifest for flatpak suport.
- removed the just module because it was installed in the final package
- put the app icon in scalable dir, and remove other because its redundant
- rename some file
- put the same appid everywhere
- fr translation
- add cargo-source.json
- simplification of the justfile

Note:

for generating the cargo source:

```
git clone https://github.com/flatpak/flatpak-builder-tools
python3 flatpak-builder-tools/cargo/flatpak-cargo-generator.py ./Cargo.lock -o cargo-sources.json
```

for building and installing

```
flatpak-builder \
        --force-clean \
        --verbose \
        --ccache \
        --user --install \
        --install-deps-from=flathub \
        --repo=repo \
        flatpak-out \
        dev.edfloreshz.CosmicTweaks.json
```

I tested and everything seems to work (expect the spacing and padding stuff but i think it unrelated because other change of the config works)

### CODE REVIEW

**Comment by wiiznokes** [comment]
note that it may be preferable to create the flatpak repo now @edfloreshz, and i can create a pull request there to avoid merging the big `cargo-sources.json`. But you can already test the flatpak with this branch.

**Comment by edfloreshz** [comment]
On second thought, could you change `dev.edfloreshz.Tweaks` to `dev.edfloreshz.CosmicTweaks`? Given that this is a COSMIC specific app, i think it would be a better fit.

Sorry about that 😅

**Comment by edfloreshz** [comment]
I'm currently traveling 🧳 , it may be a while before I can create the repository, we can hold this PR until then.

**Comment by wiiznokes** [comment]
Yeah np

**Comment by edfloreshz** [comment]
Perfect! I'll review when I'm on my laptop 💻 thanks

**Comment by edfloreshz** [comment]
I'm reviewing the PR, could you rebase it with the latest changes?

File: cargo-sources.json
Comment (edfloreshz) [inline cargo-sources.json:1]: Should this file be pushed?

File: res/metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-only</project_license>
+
+  <project_group>COSMIC</project_group>
+  <developer id="dev.edfloreshz">
+    <name>edfloreshz</name>
+  </developer>
+  <update_contact>edfloreshz@proton.me</update_contact>
+
+  <url type="homepage">https://github.com/edfloreshz/cosmic-tweaks</url>
```
Comment (edfloreshz) [inline res/metainfo.xml:22]:
```suggestion
  <url type="homepage">https://github.com/cosmic-utils/tweaks</url>
```

File: res/metainfo.xml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  <url type="bugtracker">https://github.com/edfloreshz/cosmic-tweaks/issues</url>
```
Comment (edfloreshz) [inline res/metainfo.xml:23]:
```suggestion
  <url type="bugtracker">https://github.com/cosmic-utils/tweaks/issues</url>
```

File: res/metainfo.xml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+  <launchable type="desktop-id">dev.edfloreshz.CosmicTweaks.desktop</launchable>
+
+  <icon type="remote" height="256" width="256">
+    https://raw.githubusercontent.com/edfloreshz/cosmic-tweaks/master/res/app_icon.svg</icon>
+
+  <screenshots>
+    <screenshot type="default">
+      <image>
+        https://raw.githubusercontent.com/edfloreshz/cosmic-tweaks/main/res/screenshots/window-light.png</image>
```
Comment (edfloreshz) [inline res/metainfo.xml:42]:
```suggestion
        https://raw.githubusercontent.com/cosmic-utils/tweaks/main/res/screenshots/window-light.png</image>
```

File: res/metainfo.xml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    </screenshot>
+    <screenshot>
+      <image>
+        https://raw.githubusercontent.com/edfloreshz/cosmic-tweaks/main/res/screenshots/window-dark.png</image>
```
Comment (edfloreshz) [inline res/metainfo.xml:46]:
```suggestion
        https://raw.githubusercontent.com/cosmic-utils/tweaks/main/res/screenshots/window-dark.png</image>
```

File: res/metainfo.xml [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+    <p>Customize your COSMIC desktop beyond infinity!</p>
+    <ul>
+      <li>Built with Rust for exceptional performance and responsiveness.</li>
+    </ul>
+  </description>
+
+  <launchable type="desktop-id">dev.edfloreshz.CosmicTweaks.desktop</launchable>
+
+  <icon type="remote" height="256" width="256">
+    https://raw.githubusercontent.com/edfloreshz/cosmic-tweaks/master/res/app_icon.svg</icon>
```
Comment (edfloreshz) [inline res/metainfo.xml:37]:
```suggestion
    https://raw.githubusercontent.com/cosmic-utils/tweaks/master/res/app_icon.svg</icon>
```

**Review by edfloreshz — CHANGES_REQUESTED** [review]
Some minor changes

**Comment by wiiznokes** [comment]
> I'm reviewing the PR, could you rebase it with the latest changes?

Done

File: cargo-sources.json
Comment (wiiznokes) [reply cargo-sources.json:1]: Probably not, see https://github.com/cosmic-utils/tweaks/pull/23#issuecomment-2372062956

**Comment by edfloreshz** [comment]
I'll try to make the Flathub repo tomorrow.

File: cargo-sources.json
Comment (edfloreshz) [inline cargo-sources.json:1]: Can you remove this file? I'm creating the repo but I need to merge this first.

Comment (edfloreshz) [reply cargo-sources.json:1]: You can go ahead and delete it, I already pushed it to https://github.com/edfloreshz/flathub/tree/dev.edfloreshz.CosmicTweaks

**Review by edfloreshz — APPROVED** [review]
I'll merge and delete the file later, I'm a bit short on time and I want the Flathub submission to proceed while I'm gone.

**Pull request merged**